src/routes/reference/solid-router/primitives/use-params.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: useParams
+---
+
+Retrieves a reactive, store-like object containing the current route path parameters as defined in the Route.
```

Comment:
can we briefly elaborate on the structure? eg. where does the `.id` come from in `params.id`